lib/rubocop/cop/mixin/alignment.rb

```diff
@@ -67,6 +67,10 @@ def within?(inner, outer)
 
       # @deprecated Use processed_source.comment_at_line(line)
       def end_of_line_comment(line)
+        warn Rainbow(<<~WARNING).yellow, uplevel: 1
+          `end_of_line_comment` is deprecated. Use `processed_source.comment_at_line` instead.
```

Comment:
Should this be this since that is what the methods calls itself?

```suggestion
          `end_of_line_comment` is deprecated. Use `processed_source.line_with_comment?` instead.
```